### ISSUE TITLE
Documentation out of date

### ISSUE THREAD

**Issue by proycon** [LanguageMachines/timbl#2]
Documentation still refers to the binary as ``Timbl`` instead of ``timbl``. Also, I think it should be shipped inside this repo, currently we still refer to a pdf on a server in Tilburg.

**Comment by kosloot** [comment]
Work on the Timbl  manual has been stalled for quite some time now :{
For sure it would be nice to have an updated version.

**Comment by kosloot** [comment]
Addition:
In the ILK svn repo there is an Timbl 6.4 TeX file present which mentions a lot of additions and changes , including the change for Timbl to timbl.
But the document is never fished or published, I guess.
That could be published quite easily. Bit maybe we must decide on issues like ILK versus LaMa and Tilburg vs Nijmegen first?

**Comment by proycon** [comment]
The tex source should in any case be moved to this repo in my opinion yeah.

**Comment by twocs** [comment]
The Tex documents were added but some files are missing:
fullname.sty
mble-method.eps
distanceweight-ided.eps

It's obvious from the discussion that it's still a work-in-progress, but I thought it might be helpful if I were to point out that it didn't compile even after commenting out sections above. Is there a repository where I might find those files? Or a pdf of the new version? The last one I can find is: https://ilk.uvt.nl/downloads/pub/papers/Timbl_6.3_Manual.pdf

**Comment by kosloot** [comment]
OK, these 3 missing files are added to the git repro of Timbl now.

**Comment by proycon** [comment]
Adding the latest PDF itself is probably best indeed

**Comment by kosloot** [comment]
All necessary files have been moved to the docs directory now. Including a copy of the old ILK bibtex file.
A .pdf is also included.
But we really need to overhaul the manual.

**Comment by antalvdb** [comment]
The reference guide half of this issue is addressed in #17: the manual is renamed and updated to 7.0 (`docs/Timbl_7.0_Manual.pdf` + LaTeX source), dead ilk.uvt.nl links and email addresses now point to GitHub, the install instructions are current, a 6.4→7.0 changelog documents the new `--prune` and `--limit` options, and section 5.7 adds verified 2016–2026 and cross-domain applications including the memory-based LLM work. The man page (@kosloot) and README (@proycon) bumps are tracked separately — this issue can close once those land.

**Comment by kosloot** [comment]
new docs are released with timbl 7.0
